taf/updater/updater.py

```diff
@@ -1221,16 +1186,16 @@ def _update_target_repository(
         )
     taf_logger.info("Successfully validated {}", repository.name)
 
-    if error_if_unauthenticated and len(additional_commits):
-        # these commits include all commits newer than last authenticated commit (if unauthenticated commits are allowed)
-        # that does not necessarily mean that the local repository is not up to date with the remote
-        # pull could've been run manually
-        # check where the current local head is
-        branch_current_head = repository.top_commit_of_branch(branch)
-        if branch_current_head in additional_commits:
-            additional_commits = additional_commits[
-                additional_commits.index(branch_current_head) + 1 :
-            ]
+    # if error_if_unauthenticated and len(additional_commits): #TODO: what to do with this
```

Comment:
There's a TODO left here. This is where we only leave commits which have not already been merged into the local repository (e.g. if the user manually ran pull).

Reply:
Seems like we ought to leave this in here?